code/plugins/api/articles/articles/helper/contenthelper.php

```
@@ -0,0 +1,209 @@
+require_once JPATH_SITE.'/plugins/api/articles/articles/blogs/category.php';
+
+class BlogappContentHelper 
+{
+	public $modules = array();
+
+	public $mods = array();
+
+    public function getAreadeAtuacao($idArtigo)
+    {
+
+        $jinput = Factory::getApplication()->input;
```

PHPMD (2.15.0)

Avoid unused local variables such as '$jinput'. (Unused Code Rules)

(UnusedLocalVariable)